app/uk/gov/hmrc/helptosavefrontend/views/core/confirm_check_eligibility.scala.html

```diff
@@ -18,25 +18,24 @@
 @import uk.gov.hmrc.helptosavefrontend.models.HtsContext
 @import uk.gov.hmrc.helptosavefrontend.views.html.helpers._
 @import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
+@import uk.gov.hmrc.helptosavefrontend.views.html.layout
 
-@this(main: uk.gov.hmrc.helptosavefrontend.views.html.main_template,
+@this(layout: layout,
       submit: uk.gov.hmrc.helptosavefrontend.views.html.helpers.submit)
 
 @()(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
 
-@title = @{
+@pageTitle = @{
   Messages("hts.no-account.title.h1")
 }
 
-@main(
-  title = title
-) {
-  <h1 class="heading-large">@title</h1>
+@layout(pageTitle) {
+  <h1 class="heading-large">@pageTitle</h1>
```

Comment:
this should be using classname `govuk-heading--l`

Reply:
Sure, I hope I can change <h1 class="heading-large"> to use classname .govuk-heading--l, in all these pages.